src/components/video/video.js

```diff
@@ -1,45 +1,127 @@
+/* eslint-disable jsx-a11y/media-has-caption */
 import React from 'react';
 import PropTypes from 'prop-types';
+import { VimeoPlayImage } from '../related-page/vimeo';
 
 export default class Video extends React.PureComponent {
-  render() {
-    let videoProps = {
-      autoPlay: true,
-      loop: true
+  constructor(props) {
+    super(props);
+    // by default: assume the users prefers reduced motion
+    this.state = {
+      isPlaying: false, // the video is not playing
+      autoPlay: undefined, // turn off autoplay
+      loop: undefined // turn off loop
     };
-    const reducedMotion =
+    this.video = React.createRef();
+  }
+
+  componentDidMount() {
+    // check the users preference
+    const prefersReducedMotion =
       typeof window !== 'undefined'
         ? window.matchMedia('(prefers-reduced-motion: reduce)').matches
         : false;
-    if (reducedMotion) {
-      videoProps = {
-        autoPlay: undefined,
-        loop: undefined,
-        controls: true
-      };
+    // if `prefers-reduced-motion: reduce` is not set, use props to override state defaults
+    if (!prefersReducedMotion) {
+      this.setState({
+        autoPlay: this.props.autoplay,
+        loop: this.props.loop
+      });
     }
+  }
+
+  // show video controls on hover
+  onHover = () => {
+    this.video.current.controls = true;
+  };
+
+  // hide video controls when user leaves the video container
+  onOut = () => {
+    this.video.current.controls = false;
+  };
+
+  // check if video is playing (helps us detect if user has autoplay enabled or disabled)
+  onPlay = () => {
+    this.setState({ isPlaying: true });
+  };
+
+  // check when video stops
+  onStop = () => {
+    this.setState({ isPlaying: false });
+  };
+
+  // play the video
+  playVideo = () => {
+    this.video.current.play();
+  };
+
+  render() {
+    const { src, title, muted, playsInline } = this.props;
+    const { isPlaying, autoPlay, loop } = this.state;
+
+    const videoProps = {
+      autoPlay,
+      loop,
+      muted,
+      playsInline
+    };
+
     return (
-      <div>
+      <div
+        className="relative flex-parent flex-parent--center-main flex-parent--center-cross bg-gray"
+        onMouseOver={this.onHover}
+        onMouseLeave={this.onOut}
+        onFocus={this.onHover}
+      >
+        {!isPlaying && (
+          <button
+            onClick={this.playVideo}
+            className="bg-darken75 bg-darken50-on-hover w60 h60 py12 px12 w120-mm h120-mm py18-mm px18-mm round cursor-pointer z1 absolute"
+          >
+            <VimeoPlayImage />
+          </button>
+        )}
         <video
           {...videoProps}
-          muted
           width="100%"
           className="block mx-auto"
-          src={this.props.src}
+          src={`${src}#t=0.1`} // the hash enables preview on iOS
```

Comment:
When testing on iOS, I noticed that when auto play is disabled, then the video will not show a preview (just a blank background with the play button). 

> <img src="https://user-images.githubusercontent.com/2180540/115037032-0ae81200-9e9c-11eb-9db4-ebaf7ea5348d.jpg" width="400">

I learned that iOS does this to prevent downloading the video until it's played. We have two options to generate the preview:

1. Use this hack, which tells the video player to advance the video timeline by 0.1 seconds, which will make iOS video show the preview.
2. Require that the video include an image, which the video component will use as the preview until the video starts playing/is downloaded.

Option 2 is the best practice, but it would require significant energy in rollout since we could need to generate a thumbnail image for every video (I'm thinking about android/ios examples which use a lot of video).

## What should we do?

1. Take an in between step to get us closer to the best practice: We enable this "hack" temporarily. We add a preview image option to Video so that moving forward, we will include a preview image with the video. We create an issue in documentation to quest the task of adding preview images to videos.

2. Stick with the hack. Since the blank video preview will only be seen by some iOS users who have autoplay disabled - this is likely a small fraction of our users.

3. Don't do the hack, do not add preview images (but add an optional preview image prop to make the component future friendly), and instead set a background color (or figure out a very generic preview image) to the component define the video area to prevent it from looking blank.

4. Something else?

@colleenmcginnis @danswick any opinions? I'm leaning toward number 3, since this issue likely effects very few users and while I strive for best practices, adding preview images to all videos would be a significant undertaking.

Reply:
@katydecorah I'm good with option 3! I think just a background color (that has good contrast with the play button) would be enough to suggest that there will be a video there even if it's not ready at the start.

Reply:
Agreed! I think option 3 is a good place to start and we can revisit if it seems to be causing issues. Also agreed that just a background color is probably totally sufficient 👍

Reply:
Thanks for the feedback! I'm going to remove the hack and add a background to this component.